### PR TITLE
feat: 🎸 Support fetching historical instructions for a venue

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

Allows fetching historical instructions for a venue using middleware v2 (native Subquery)

### Breaking Changes

NA

### JIRA Link

DA-167

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/utils/conversion.ts [hunk 1]
```diff
@@ -4365,3 +4368,59 @@ export function metadataValueDetailToMeshMetadataValueDetail(
...
+    typeDetails = {
+      type: InstructionType.SettleOnAffirmation,
+    };
+  }
+
+  return {
+    id: new BigNumber(instructionId),
+    blockNumber: new BigNumber(blockId),
+    blockHash: hash,
+    status: status as InstructionStatusEnum,
```
Comment (polymath-eric) [inline src/utils/conversion.ts:4411]: (nit) I think this works. If not OK to leave

```suggestion
    status: status,
```

**Review by polymath-eric — APPROVED** [review]

File: src/utils/conversion.ts (hunk 1 above)
Comment (prashantasdeveloper) [reply src/utils/conversion.ts:4411]: I have updated the same in the next PR

**Comment by prashantasdeveloper** [comment]
I would like to wait to merge this until the parent PR is merged, so that these would trigger a separate release for each feat

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**